sentry_sdk/utils.py

```diff
@@ -1126,6 +1144,74 @@ def from_base64(base64_string):
     return utf8_string
 
 
+Components = namedtuple("Components", ["scheme", "netloc", "path", "query", "fragment"])
+
+
+def sanitize_url(url):
+    # type: (str) -> str
+    """
+    Removes all query parameter values and username:password from a given URL.
+    """
+    parsed_url = urlsplit(url)
+    query_params = parse_qs(parsed_url.query, keep_blank_values=True)
+
+    from sentry_sdk.consts import SENSITIVE_DATA_SUBSTITUTE
```

Comment:
why is the import inside?

Reply:
Because of a circular import problem: https://github.com/getsentry/sentry-python/actions/runs/4183447955/jobs/7247820509

This is currently happening when we create our API docs, so in the future we have to decide what to do with API docs, because right now they are kind of abandoned.

Reply:
i think you can move it to the bottom of the file then

Reply:
lets try

Reply:
Moved the variable to `utils.py` now because it is mainly used there to fix the circular import.